### PR TITLE
termination, hibernate, stop instance 

### LINKED ISSUE #27: AWS services shutdown
- [x] Research how to shutdown AWS service
- [x] Create service for AWS services shutdown

### PR DESCRIPTION
**Pull request by guser21**
Done for EC2, RDS, EBS 
closes #27

### CODE REVIEW

**Comment by mbalc** [comment]
@guser21 I just heard about that feature about a week ago or so, check this out
https://laravel-news.com/github-adds-draft-pull-requests

File: server/src/main/java/com/omigost/server/aws/NewBudgetRequestBuilder.java [hunk 1]
```diff
@@ -6,7 +6,9 @@
 import org.springframework.util.MultiValueMap;
 
 import java.math.BigDecimal;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
```
Comment (mbalc) [inline server/src/main/java/com/omigost/server/aws/NewBudgetRequestBuilder.java]: Why this change? Should I avoid such implicit imports in my code?

File: server/src/main/java/com/omigost/server/rest/dto/TerminationRequest.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.omigost.server.rest.dto;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class TerminationRequest {
+    public List<String> machineId;
```
Comment (mbalc) [inline server/src/main/java/com/omigost/server/rest/dto/TerminationRequest.java]: could we rename to `machineIds`?

File: server/src/main/java/com/omigost/server/aws/NewBudgetRequestBuilder.java (hunk 1 above)
Comment (guser21) [reply server/src/main/java/com/omigost/server/aws/NewBudgetRequestBuilder.java]: No I just formatted the code and automagically it changed

File: server/src/main/java/com/omigost/server/rest/dto/TerminationRequest.java (hunk 2 above)
Comment (guser21) [reply server/src/main/java/com/omigost/server/rest/dto/TerminationRequest.java]: sure

**Comment by Ogochi** [comment]
Please mark in issue what have u done. @guser21
